File: Python/0218_2.py
def sequence_sum(begin_number, end_number, step):
    return sum(range(begin_number, end_number + 1, step))

# 範例
print(sequence_sum(12, 2, 2))   # 輸出: 0
print(sequence_sum(2, 6, 2))   # 輸出: 12
print(sequence_sum(1, 5, 1))   # 輸出: 15
print(sequence_sum(1, 5, 3))   # 輸出: 5

File: JavaScript/0218_2.js
function sequenceSum(begin, end, step) {
    if (begin > end) {
      return 0;
    }
  
    const count = Math.floor((end - begin) / step) + 1;
    return (begin + end) * count / 2;
  }

// 範例
console.log(sequenceSum(12, 2, 2));   // 輸出: 0
console.log(sequenceSum(2, 6, 2));   // 輸出: 12
console.log(sequenceSum(1, 5, 1));   // 輸出: 15
console.log(sequenceSum(1, 5, 3));   // 輸出: 5